### ISSUE TITLE
React cell editor swallows first keystroke

### ISSUE THREAD

**Issue by grovesNL** [ag-grid/ag-grid-react#84]
**I'm submitting a ...**
```
[X] bug report => search github for a similar issue or PR before submitting
[X] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid-react/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
When a React cell editor component is provided, the first keystroke is lost (entering cell editor mode). There needs to be a hook to include this first keystroke rather than consumers attempting to capture `window.event` or something similar.

An example is on [ag-grid's example-react-editor page](https://www.ag-grid.com/example-react-editor/#gsc.tab=0). If you start typing a number, say `1`, in the numeric column, then the first key press is swallowed and you won't see the `1` appear anywhere in the rendered cell editor.

**Expected behavior**
The first keystroke is treated identically to the editor as subsequent keystrokes. In the example, I would expect `1` to be rendered if I entered the cell editor by pressing `1`.

**Minimal reproduction of the problem with instructions**
Go to (https://www.ag-grid.com/example-react-editor/#gsc.tab=0 and start typing in the numeric column, notice that the first key press is swallowed and not exposed to the component.

**What is the motivation / use case for changing the behavior?**
Users expect the same behavior with the React cell editor as they do with the regular ag-grid cell editor.

**Comment by seanlandsman** [comment]
Hi,

Thanks for raising this. I've raised AG-701 our side to track this - you can view its progress on our pipeline page here: https://www.ag-grid.com/ag-grid-pipeline/

We'll update this thread with updates as soon as we have any.

thanks

**Comment by seanlandsman** [comment]
I've made a tweak to this example so that it doesnt lose the first key - thanks

**Comment by grovesNL** [comment]
@seanlandsman Thanks, what was the change you made? Are the examples on github so I could look at the diffs myself in the future?

**Comment by JohnMasterson** [comment]
Just to confirm that this was included in version 14.0.0 which is now available.

Regards, 

John
